build_overrides/webnn_features.gni

```diff
@@ -31,4 +31,6 @@ declare_args() {
   # Enables the compilation of WebNN's Null backend
   # (required for unittests, obviously non-conformant)
   webnn_enable_null = true
+
+  webnn_enable_wire = false
```

Comment:
should we turn on wire support by default?

Reply:
Some tests are failed that need need more investigation but current status doesn't block chromium integration. I plan to go ahead to enable it in chromium.

Reply:
It makes sense. Before we merge this, could you please file an issue about the failures? So others may hlep fix them.

Reply:
File the issue https://github.com/otcshare/webnn-native/issues/927